arrays/arrays.c

```diff
@@ -21,12 +22,15 @@ typedef struct Array {
  * Allocate memory for a new array
  *****/
 Array *create_array (int capacity) {
-  // Allocate memory for the Array struct
-
-  // Set initial values for capacity and count
-
-  // Allocate memory for elements
-
+  //malloc for Array Struct
+  Array *arr = malloc(sizeof(Array));
+  //init capacity and count
+  arr->capacity = capacity;
+  arr->count = 0;
+  //malloc for elements
+  arr->elements = malloc(capacity * sizeof(char *));
```

Comment:
nice job making sure to include `sizeof(char *)`, not everybody got this the first time!